src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -6,6 +6,7 @@
     <TargetFrameworkNetStandard>netstandard2.0</TargetFrameworkNetStandard>
     <TargetFrameworkNet6>net6.0</TargetFrameworkNet6>
     <TargetFrameworkNet6Win>net6.0-windows</TargetFrameworkNet6Win>
+    <TargetFrameworkUap>uap10.0.17763</TargetFrameworkUap>
```

Comment:
Can you please add this to the "legacy" property group (line 13). We should not build this by default

Reply:
updated